recipes/libtiff/all/conanfile.py

```diff
@@ -70,19 +70,9 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            try:
-                del self.options.fPIC
-            except Exception:
-                pass
-        if not self.options.cxx:
```

Comment:
@SpaceIm I can't tell at the moment if removing this line is OK or not... Wouldn't you only want to remove compiler.cppstd and compiler.libcxx if the cxx option is not set?

Reply:
oops correct, I'll fix that